release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -578,6 +578,12 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-6-deprecated-features"]
 === Deprecated features
 
+[discrete]
```

Comment:
I don't think it's necessary to add the `[discrete]` tag, since it's four levels deep. I believe we only use that tag under `Notable technical changes`, since we skip from level 2 -> 4 in that section (those display the level four headers in the TOC since level three is skipped). Did you find it was necessary?

Reply:
@codyhoag Good catch - I initially had started drafting this in the notable technical changes section, which is why it's there. I'll remove it.